editors/vim/Portfile

```diff
@@ -3,6 +3,10 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
+# The vim port should always follow the patch level used in the MacVim port.This
+# is because they both share the same set of configuration files and ensures
+# that the user's configuration files will always work with both.
+# https://github.com/macvim-dev/macvim/releases
```

Comment:
I think we don't need the URL here, because whoever wants to update is supposed to look at the MacVim port.